src/wallet/coinselection.cpp

```diff
@@ -279,13 +279,13 @@ bool KnapsackSolver(const CAmount& nTargetValue, std::vector<OutputGroup>& group
         }
 
         if (LogAcceptCategory(BCLog::SELECTCOINS)) {
-            LogPrint(BCLog::SELECTCOINS, "SelectCoins() best subset: "); /* Continued */
+            std::string log_message{"SelectCoins() best subset: "};
```

Comment:
Not introduced in this change, but this function name is wrong. Should be `KnapsackSolver()`? Or simply `__func__`.

Reply:
I'll let @achow101 decide

Reply:
`__func__`

Reply:
On a second thought, it is already possible to log the function name with `-logsourcelocations`, so maybe just remove it?

Reply:
Resolving discussion for now. Let me know if anything needs to be changed after the latest commit fad8481